include/swift/Subsystems.h

```diff
@@ -249,18 +252,22 @@ namespace swift {
 
   /// Turn the given Swift module into either LLVM IR or native code
   /// and return the generated LLVM IR module.
+  /// If you set an outModuleHash, then you need to call performLLVM.
```

Comment:
This seems too subtle for me. Can we just *always* require callers to manually call performLLVM?